### PR TITLE
Update tab-view.md

### PR DESCRIPTION
**Pull request by jofftiquez**

### CODE REVIEW

File: content/docs/en/elements/components/tab-view.md [hunk 1]
```diff
@@ -51,7 +51,7 @@ contributors: [MisterBrownRSA, rigor789, eddyverbruggen, ikoevska, kharysharpe]
 
 | Name | Description |
 |------|-------------|
-| `selectedIndexChange` | Emitted when one of the `<TabViewItem>` components is tapped.
+| `selectedIndexChange` | Emitted when one of the `<TabViewItem>` components was tapped.
```
Comment (ikoevska) [inline content/docs/en/elements/components/tab-view.md]:
```suggestion
| `selectedIndexChange` | Emitted when one of the `<TabViewItem>` component is tapped.
```

**Comment by ikoevska** [comment]
This seems to be a duplicate of https://github.com/nativescript-vue/nativescript-vue.org/pull/210 with https://github.com/nativescript-vue/nativescript-vue.org/pull/210 building on the existing information.

**Comment by rigor789** [comment]
Closing in favor of #210

**Comment by jofftiquez** [comment]
Alright!

**Pull request closed without merging**